runtime/compiler/env/VMJ9.cpp

```diff
@@ -2439,6 +2439,27 @@ TR_J9VMBase::isHotReferenceFieldRequired()
    return TR::Compiler->om.isHotReferenceFieldRequired();
    }
 
+bool
+TR_J9VMBase::isIndexableDataAddrPresent()
+   {
+#if defined(J9VM_ENV_DATA64)
+   return FALSE != _jitConfig->javaVM->isIndexableDataAddrPresent;
+#else
+   return false;
+#endif /* defined(J9VM_ENV_DATA64) */
+   }
+
+/**
+ * Query if off-heap large array allocation is enabled
+ *
+ * @return true if off-heap large array allocation is enabled, false otherwise
+ */
+bool
+TR_J9VMBase::isOffHeapAllocationEnabled()
+   {
+   return TR::Compiler->om.isOffHeapAllocationEnabled();
+   }
+
```

Comment:
Does JIT still need these functions? If yes, we can merge this piece separately earlier.

Reply:
@rmnattas @zl-wang please move this into a JIT PR; does not seem like it depends on any new changes in this PR

Reply:
Opened a PR for it https://github.com/eclipse-openj9/openj9/pull/20187/files